namespaces/init.go

```diff
@@ -141,6 +151,85 @@ func Init(container *libcontainer.Config, uncleanRootfs, consolePath string, pip
 	return system.Execv(args[0], args[0:], os.Environ())
 }
 
+func initUserNs(container *libcontainer.Config, uncleanRootfs, consolePath string, pipe *os.File, args []string) (err error) {
```

Comment:
I'm asking because the look pretty similar to each other, maybe we can just wrap unneeded parts in 
```
if container.Namespaces.Contains(libcontainer.NEWUSER) {
}
```